src/components/Footer.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import { TodoFilter } from './TodoFilter';
+import { TodoClearCompleted } from './TodoClearCompleted';
+import { TodoActiveCount } from './TodoActiveCount';
+import { TodosContext } from '../context/TodosContext';
+
+type PropsFooter = {
+};
+
+export const Footer: React.FC<PropsFooter> = () => {
+  const { todos } = useContext(TodosContext);
+  const completedTasksCount = todos.filter(todo => todo.completed).length;
+
+  return (
+    <footer className="footer">
+      <TodoActiveCount />
+
+      <TodoFilter />
+
+      {completedTasksCount !== 0 && (
```

Comment:
```suggestion
      {!!completedTasksCount && (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**